receiver/mongodbreceiver/metadata.yaml

```diff
@@ -146,3 +148,111 @@ metrics:
       value_type: int
       monotonic: true
     attributes: [database]
+  mongodb.database.count:
+    description: The number of existing databases.
+    unit: "{databases}"
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+    attributes: []
+  mongodb.index.access.count:
+    description: The number of times an index has been accessed. 
+    unit: "{accesses}"
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+    attributes: [database, collection]
+  mongodb.document.insert.count:
+    description: The number of documents inserted.
+    unit: "{documents}"
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+    attributes: [database]
+  mongodb.document.delete.count:
+    description: The number of documents deleted.
+    unit: "{documents}"
+    enabled: true
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+    attributes: [database]
+  mongodb.document.update.count:
```

Comment:
`inserts`, `deletes`, `updates`?

Should these just be an attribute on a `mongodb.document.operation.count` metric?

Reply:
updated.